### DISCUSSION TITLE
Scalability

### DISCUSSION THREAD

**Discussion by ShreyasTitus** [divyagiridhar/SE_Project_2#16] [Ideas]
We do not need to implement scalability based on the information posted on discord. If we implement it, it would be good but if we discuss a way to implement it, that would be great for us as well

**Comment by ShreyasTitus** [comment]
We are able to scale things up with AWS EC2 instances, @divyagiridhar and I can work on it

**Comment by divyagiridhar** [comment]
Yeah let's start writing the documentation for our plan

**Comment by ShreyasTitus** [comment]
Hey @divyagiridhar could you upload the scalability document which we worked on?

**Reply by divyagiridhar** [reply]
Done
